From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Hide Hako <opi5plus@bcc.bai.ne.jp>
Date: Tue, 26 Aug 2025 01:44:00 +0000
Subject: FROMLIST(v2): arm64: dts: rockchip: Fix sound output from the audio
 jack on OrangePI5 Plus

Currently, analog sound is not output from the audio jack.
This patch allows you to select analog headphones in alsamixer.

Fixes: 236d225e1ee7 ("arm64: dts: rockchip: Add board device tree for rk3588-orangepi-5-plus")
Signed-off-by: Hide Hako <opi5plus@bcc.bai.ne.jp>
---
 arch/arm64/boot/dts/rockchip/rk3588-orangepi-5-plus.dts | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/boot/dts/rockchip/rk3588-orangepi-5-plus.dts b/arch/arm64/boot/dts/rockchip/rk3588-orangepi-5-plus.dts
index 111111111111..222222222222 100644
--- a/arch/arm64/boot/dts/rockchip/rk3588-orangepi-5-plus.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3588-orangepi-5-plus.dts
@@ -78,6 +78,7 @@ &analog_sound {
 	pinctrl-0 = <&hp_detect>;
 	simple-audio-card,aux-devs = <&speaker_amp>, <&headphone_amp>;
 	simple-audio-card,hp-det-gpios = <&gpio1 RK_PD3 GPIO_ACTIVE_HIGH>;
+	simple-audio-card,pin-switches = "Speaker", "Headphones";
 	simple-audio-card,widgets =
 		"Microphone", "Onboard Microphone",
 		"Microphone", "Microphone Jack",
-- 
Armbian

